### ISSUE TITLE
Deploy

### ISSUE THREAD

**Issue by octaviancernat** [ploneintranet/ploneintranet#911]
Hi everyone!
I'm new to plone and ploneintranet as well. Ploneintranet looks very good, congratulations.
For the past four days I've been testing your product, but I have some problems.
 
1. After deploy is there any order to install the modules? I mean, like plone modules first, ploneintranet second?
2. I dont know if a did something wrong, but I can't add Library. In frontend the  "Create a new page or section" seems to have the wrong url. It points to @@switch_theme so it does nothing. This

I tried adding library from "/library/manage_main" with "Dexterity container" but it redirects me to the frontend.
I tried with "library/@@add_folder", the form shows but it's not saving. It throws a js error.
I can add content instead with "library/human-resources/leave-policies/@@add_content".

Can you please point me in the right direction with this? I could really use your help with this.

Thank you.

**Comment by pysailor** [comment]
Hi @octaviancernat ,

1. about the installation:

 In case you have already tried http://docs.quaive.net/installation/quickstart.html#create-a-new-plone-intranet-instance
I usually install it like this. I add a new Plone site. Then, from Plone, I install Plone Intranet [suite base]:

<img width="889" alt="screen shot 2018-06-20 at 17 30 10" src="https://user-images.githubusercontent.com/642660/41668705-3de336ee-74b0-11e8-9bb9-c8a51d0ca329.png">

Then I go to the ZMI -> portal_setup -> Import and pick "Plone Intranet: Suite"
<img width="783" alt="screen shot 2018-06-20 at 17 33 44" src="https://user-images.githubusercontent.com/642660/41668753-6188eb84-74b0-11e8-90a7-5541e07dbba9.png">

(-> import all steps)

Then you should be able to access your ploneintranet site. (And the create or import users in ploneintranet, so that you can log in with a proper user, not the Zope user for the ZMI).

2. About the library: 

Currently, adding content is not supported via the themed UI from ploneintranet. Therefore the skin switcher is present so that for the library, you can switch to the default Plone view (barceloneta). See http://docs.quaive.net/development/frontend/themeswitcher.html about how to configure it.
Basically, you will need 2 different addreses like my.ploneintranet.com and cmsmy.ploneintranet.com. The first one is default address for ploneintranet. It needs to go into this registry entry:
http://localhost:8080/Plone/portal_registry/edit/ploneintranet.themeswitcher.interfaces.IThemeSwitcherSettings.hostname_default
The CMS address will be for the plain Plone view. It needs to go into this setting of portal_registry:
http://localhost:8080/Plone/portal_registry/edit/ploneintranet.themeswitcher.interfaces.IThemeSwitcherSettings.hostname_switchlist

More about adding content to the library can be found here: http://manual.quaive.net/library/index.html#adding-content-to-the-library

Hope that helps!
Cheers,
Wolfgang

**Comment by octaviancernat** [comment]
Hi @pysailor,
Thank you for taking your time to write this guide. It seems I got lost in the documentation :).
I got it working on 2 machines. I got an empty instance with default config an one with the Testing Content same with default config

Everything works well except the news app. I can add article and hero image. I can't add or edit any content. 
On both instances i get this error on save action:

> Plone/news/fames-vitae/edit.html
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module ploneintranet.workspace.basecontent.baseviews, line 167, in __call__
  Module ploneintranet.news.browser.publisher, line 213, in update
  Module ploneintranet.workspace.basecontent.baseviews, line 276, in update
  Module <decorator-gen-256>, line 2, in transition
  Module plone.api.validation, line 70, in wrapped
  Module <decorator-gen-255>, line 2, in transition
  Module plone.api.validation, line 131, in wrapped
  Module <decorator-gen-254>, line 2, in transition
  Module plone.api.validation, line 100, in wrapped
  Module plone.api.content, line 475, in transition
InvalidParameterError: Invalid transition 'published'.
Valid transitions are:
reject
retract

It doesn't matter the state it has the error will occur on each save.
